client/my-sites/plugins/plugin-details-CTA/preinstalled-premium-plugins-CTA.jsx

```diff
@@ -53,10 +53,12 @@ export default function PluginDetailsCTAPreinstalledPremiumPlugins( {
 						isFetching ? (
 							<div className="plugin-details-CTA__price-placeholder">...</div>
 						) : (
-							<>
-								{ price + ' ' }
-								<span className="plugin-details-CTA__period">{ period }</span>
-							</>
+							translate( '{{span}}From{{/span}} %(price)s {{span}}%(period)s{{/span}}', {
```

Comment:
The "From" part seems specific to the Jetpack Search plugin only. Are we use we want to always display that on all preinstalled premium plugins?

Reply:
Yeah, looking at how Checkout handles things, it might be an exception for JP Search — for now at least.

I'm not very comfortable adding an exception for 1 single product, but in this case it wouldn't introduce much debt, and it'd move this PR forward.